### ISSUE TITLE
Request chromecast for multiple apps

### ISSUE THREAD

**Issue by FPSUsername** [AndreyPavlenko/Fermata#302]
Last time I tried to figure out how to get F1 TV (via the app) to play on AA. I saw that the chromecast setting is present in the fermata auto app, but it doesn't seem to show up for nearby devices.

The construction would be as following with two phones.
Phone 1 connects to AA and uses Fermata Auto.
Phone 2 has any chromecast compatible app running (e.g. F1 TV) and casts to the other device.

I'm not sure if both devices need to be connected to a same WiFi network for this to work though.

Or maybe there's a better way to integrate other apps? (perhaps this is going off topic for this app).

**Comment by AndreyPavlenko** [comment]
The app implements a chromecast sender, but not a receiver. Implementing a receiver is much more complicated due to the authentication issue - https://github.com/thibauts/node-castv2/issues/2

**Comment by fwon2638** [comment]
There is an app called "Local Video Player" https://www.youtube.com/watch?v=fsUIwqlHg94

Is there something you can implement?

**Comment by fwon2638** [comment]
Yes, but video selection is done through/from the phone (rather than from the AA launcher). So, I am wondering if you can implement a similar method (ie making selection from phone) when choosing videos saved in the phone or (more ideally) selecting Youtube video from the phone?

**Comment by AndreyPavlenko** [comment]
You can do with Fermata the same. Open Fermata on AA and the phone and start playback from the phone. It doesn't work for youtube.
